Documentation/debugging/using-your-build.md

```diff
@@ -0,0 +1,70 @@
+Using Your Build
+================
+
+This document explains how to use your CoreFX binaries in a .NET Core console app after you've built the repo.
+
+## Overview
+
+You will create a self-contained .NET Core app via `dotnet publish`, which will include stock images of framework assemblies. Then, you will replace some of those assemblies with ones from your CoreFX build.
+
+## Prerequisites
+
+- .NET Core tools - Install at https://www.microsoft.com/net/core
+
+## Steps
+
+0. Create the .NET Core app you want to use with your CoreFX build.
+
+```sh
+mkdir DotnetLab
+cd DotnetLab
+dotnet new
+```
+
+1. Choose a runtime you would like to target your app against.
+
+- A runtime (not to be confused with the [CoreCLR runtime](https://github.com/dotnet/coreclr)) is a specific architecture and OS your .NET app will target.
+
+- You must choose a runtime for your app because although .NET is platform-agnostic, in order for your app to be self-contained it must include the CoreCLR runtime, which is written in native code and platform-dependent.
+
+- Examples of valid *runtimes identifiers (RIDs)* are `win10-x64` and `win10-x86`. You can find more at the [RID catalog](https://docs.microsoft.com/en-us/dotnet/core/rid-catalog).
+
+- Make sure you have built CoreFX for this runtime. If you aren't sure, check `<corefx root>/bin/runtime` and see if there is a folder corresponding to that runtime.
+
+2. Restore your app's dependencies.
+
+```sh
+dotnet restore -r <runtime>
+```
+
+3. Create a self-contained image of your app.
+
+```sh
+dotnet publish -r <runtime>
+```
+
+4. Replace the stock framework assemblies from `dotnet publish` with your CoreFX build.
+
+```sh
+# Unix
+source=<corefx root>/bin/runtime/netcoreapp-<OS>-<configuration>-<architecture>
+destination=<project root>/bin/<configuration>/<framework>/<runtime>/publish
+cp -rT "$source" "$destination"
```

Comment:
I did this in case `bin/runtime/` includes subdirectories which need to be copied in the future. I have little scripting experience in PS so I assumed `Copy-Item $source/* $dest` would handle recursively copying directories, but it looks like I was wrong. Both versions can probably stay as they are until one of them is broken.